vms/platformvm/warp/message/register_l1_validator_test.go

```diff
@@ -52,24 +52,24 @@ func TestRegisterSubnetValidator(t *testing.T) {
 	var expectedValidationID ids.ID = hashing.ComputeHash256Array(bytes)
 	require.Equal(expectedValidationID, msg.ValidationID())
 
-	parsed, err := ParseRegisterSubnetValidator(bytes)
+	parsed, err := ParseRegisterL1Validator(bytes)
 	require.NoError(err)
 	require.Equal(msg, parsed)
 }
 
-func TestRegisterSubnetValidator_Verify(t *testing.T) {
-	mustCreate := func(msg *RegisterSubnetValidator, err error) *RegisterSubnetValidator {
+func TestRegisterL1Validator_Verify(t *testing.T) {
+	mustCreate := func(msg *RegisterL1Validator, err error) *RegisterL1Validator {
 		require.NoError(t, err)
 		return msg
 	}
 	tests := []struct {
 		name     string
-		msg      *RegisterSubnetValidator
+		msg      *RegisterL1Validator
 		expected error
 	}{
 		{
 			name: "PrimaryNetworkID",
-			msg: mustCreate(NewRegisterSubnetValidator(
+			msg: mustCreate(NewRegisterL1Validator(
 				constants.PrimaryNetworkID,
 				ids.GenerateTestNodeID(),
 				newBLSPublicKey(t),
```

Comment:
My understanding is that this is testing the `RegisterL1Valdiator` message, not the pre-existing `AddSubnetValidatorTx`. This specific test case checks that it's not valid for a `ReigsterL1Validator` message to set the subnet ID to the primary network ID, since it cannot be used for adding primary network validators.